source/developers-guide/backend-components/listing/index.md

```diff
@@ -0,0 +1,532 @@
+            columns.push(column);
+        }
+    }
+
+    me.fireEvent(me.eventAlias + '-after-create-columns', me, columns);
+
+    return columns;
+}
+```
+
+Um die neue Spalte hinzuzufügen muss im Override Beispiel die `createColumns()` Funktion überschrieben werden. Im Event System Beispiel kann das Event `product-before-create-action-columns` verwendet werden:
+
+**Mittels Override**
+```javascript
+Ext.define('Shopware.apps.SwagProduct.view.list.Product', {
+    extend: 'Shopware.grid.Panel',
+    alias:  'widget.product-listing-grid',
+    region: 'center',
+    
+    createColumns: function() {
+        var me = this,
+            columns = me.callParent(arguments);
+
+        var column = {
+            xtype: 'gridcolumn',
+            header: 'Created in july',
+            renderer: me.columnRenderer,
+            sortable: false,
+            dataIndex: 'inJuly'
+        };
+
+        columns = Ext.Array.insert(
+            columns,
+            columns.length - 1,
+            [ column ]
+        );
+
+        return columns;
+    },
+
+    columnRenderer: function(value, metaData, record) {
+        var date = record.get('createDate');
+        return this.booleanColumnRenderer((date.getMonth() === 6));
+    }
+});
+```
+
+**Mittels Event System**
+```javascript
+Ext.define('Shopware.apps.SwagProduct.controller.Main', {
+    extend: 'Enlight.app.Controller',
+    init: function() {
+        var me = this;
+        me.control({
+            'product-listing-grid': {
+                'product-before-create-action-columns': me.addColumn
+
+            }
+        });
+        me.mainWindow = me.getView('list.Window').create({ }).show();
+    },
+
+    addColumn: function(grid, columns) {
+        var me = this;
+
+        columns.push({
+            xtype: 'gridcolumn',
+            header: 'Created in july',
+            renderer: me.columnRenderer,
+            sortable: false,
+            dataIndex: 'inJuly'
+        });
+
+        return columns;
+    },
+
+    columnRenderer: function(value, metaData, record) {
+        var date = record.get('createDate');
+        return this.booleanColumnRenderer((date.getMonth() === 6));
+    }
+});
+```
+
+In diesem Beispiel ist eine Besonderheit enthalten. Und zwar ist die Funktion `booleanColumnRenderer`. Diese Funktion wird in beiden Beispielen verwendet ohne dass diese im Shopware.grid.Panel noch im Enlight.app.Controller definiert wurde. Hierbei handelt es sich um eine Helfer Funktion von Shopware, welche sich in der `Shopware.model.Helper` <b>Klasse</b> befindet. Diese <b>Klasse</b> wird in allen Komponenten per Ext JS mixin eingebunden. Dadurch stehen sämtliche Funktionen dieser Klasse im `this` Scope zur Verfügung. Die `booleanColumnRenderer` Funktion im Main Controller Beispiel steht zur Verfügung da die Renderer Funktion als Scope das Grid Panel übergeben bekommt.
+
+
+### Plugin Download - [SwagProduct.zip](http://community.shopware.com/files/downloads/swagproduct-14067634.zip)
```

Comment:
Das Plugin bitte in das Repository unter example plugins committen. Das wird dann automatsich als zip gepackt. Siehe README.md